test/server/cardImplementations/01_SOR/2-1BSurgicalDroid.spec.js

```diff
@@ -0,0 +1,54 @@
+const exp = require('constants');
+
+describe('2-1B Surgical Droid', function() {
+    integration(function() {
+        describe('2-1B Surgical Droid\'s healing', function() {
+            beforeEach(function () {
+                this.setupTest({
+                    phase: 'action',
+                    player1: {
+                        groundArena: [
+                            { card: '21b-surgical-droid', exhausted: false },
+                            { card: 'r2d2#ignoring-protocol', damage: 1 }],
+                    },
+                    player2: {
+                        groundArena: ['wampa']
+                    }
+                });
+
+                this.surgicalDroid = this.player1.findCardByName('21b-surgical-droid');
+                this.r2d2 = this.player1.findCardByName('r2d2#ignoring-protocol');
+                this.wampa = this.player2.findCardByName('wampa');
+
+                this.p1Base = this.player1.base;
+                this.p2Base = this.player2.base;
+
+                this.noMoreActions();
+            });
+
+            it('surgical droid should heal r2d2 to full', function () {
+                expect(this.r2d2.damage).toBe(1);
```

Comment:
My weak preference is that we don't add expectations which are basically just checking that the test infrastructure is working (if I've left code like that in other tests please lmk and I'll remove it).

If you would prefer to keep some tests like this in for redundancy though I'm open to it